courses/fundamentals_of_ada/020_declarations.rst

```diff
@@ -155,6 +155,24 @@ Pragmas
    pragma Page;
    pragma Optimize ( Off );
 
+------
+Quiz
+------
+
+Which statement is legal?
+
+   A. Function : constant := 1;
```

Comment:
All set. RST is more limiting than I would like (this would look much better if we used pure LaTeX) but it looks better now.
(Added a second role "answermono" so we can make all choices monospace (when appropriate)